mobile_verifier/migrations/16_modeled_coverage.sql

```diff
@@ -0,0 +1,29 @@
+CREATE TYPE signal_level AS ENUM (
+       'none',
+       'low',
+       'medium',
+       'high'
+);
+
+CREATE TABLE hex_coverage (
+       uuid UUID NOT NULL,
+       hex BIGINT NOT NULL,
+       indoor BOOLEAN NOT NULL,
+       cbsd_id TEXT NOT NULL,
+       signal_level signal_level NOT NULL,
+       coverage_claim_time TIMESTAMPTZ NOT NULL,
+       inserted_at TIMESTAMPTZ NOT NULL,
+       PRIMARY KEY (uuid, hex)
+);
+
+CREATE TABLE seniority (
+       cbsd_id TEXT NOT NULL,
```

Comment:
note for when it comes around, but wifi hotspots do not have a cbsd_id and then not null wont be compatible